css/large.css

```diff
@@ -0,0 +1,18 @@
+#hang {
+  position: absolute;
+  width: 100px;
+  height: 20px;
+  right: 0;
+  bottom: -10px;
+  background-color: #f72210;
+  z-index: 2;
+  clip-path: polygon(0% 0%, 100% 0%, 100% 100%, 10% 100%);
+}
+
+#head-container {
+  position: relative;
```

Comment:
`SUGGESTION`

Consider using the `position-relative` Bootstrap class name for this selector element and in all other occurrences.

Reply:
Hi Bola, am trying to make the position attribute responsive. that is i want to make it relative on large screen and fixed on small screen and from bootstrap documentation,the position attribute is not responsive that is why it is like that.

Reply:
Okay, Nice :+1: